robotbase/src/main/java/com/team2357/frc2022/arduino/RobotArduino.java

```diff
@@ -0,0 +1,40 @@
+package com.team2357.frc2022.arduino;
+
+import com.team2357.frc2022.Constants;
+import com.team2357.lib.arduino.ArduinoUSBController;
+
+public class RobotArduino extends ArduinoUSBController{
+
+    public RobotArduino(String ttyDevice) {
+        super(ttyDevice);
+        super.start();
```

Comment:
I have concerns about starting up a USB serial connection in the constructor here, which is in turn called in the RobotContainer() constructor. We should probably talk about how to ensure the Arduino connection is resilient and test a few scenarios. (like disconnecting and reconnecting the USB cable while it's running, to ensure we can reconnect automatically)

Reply:
I did test to see if the changes made in this PR worked, and we did have USB at this point.